asmCrashReport.pri

```diff
@@ -15,6 +15,14 @@ CONFIG (release, release|debug) {
         $$PWD/src/asmCrashReport.cpp
 
     win32-g++* {
+
+        CONFIG += file_copies
+        ADDR2LINE = $$system(where addr2line)
+        # https://stackoverflow.com/a/54162789
+        addr2line.files = $$ADDR2LINE
+        addr2line.path = $$OUT_PWD/release/tools
+        COPIES += addr2line
+
```

Comment:
`win32-g++` might be cygwin or it might be MinGW. (I haven't tried building with MinGW, but I assume it would work :-) )

Additionally, `where addr2line` will only work if the cygwin bin directory is in your PATH.

If you want to make this PR work, then you would need to:

- determine if you are building with cygwin
- determine if `addr2line` is installed
- copy all the required files (see README), not just the exe (otherwise it won't work on a target machine without cygwin)

Reply:
I assumed library user using prebuild Qt libraries. With `win32-g++`, prebuilt comes with MinGW.
You can decide what to do. This is just a suggestion. We can cancel this PR.

Also, this did not work my situation. I copied to my DESTDIR at pro file.

Reply:
Ah! I see - my apologies. I'm just in my morning-confusion state and am mixing up what I'm doing with cygwin and MinGW. (I haven't looked at this in a long time.)

Is the `addr2line` from MinGW statically built? And included with the Qt binary distribution? If so, then we would just need to:

- determine if `addr2line` from MinGW exists (again - I don't think 'where' is correct since it might not be in the path)
- copy the exe

If it's not statically built, then we would need to add all the required DLLs as well.

Reply:
Nope.

- After install the Qt 5.12 I found it at below. Also, QtCreator appends MinGW bin to the path :
  - `C:\qt\Tools\mingw730_32\bin\addr2line.exe`
  - `C:\qt\Tools\mingw730_64\bin\addr2line.exe`
- Checked with dependency walker and below the dependency.
  - `kernel32.dll`
  - `msvcrt.dll`
  - `user32.dll`

Reply:
I feel like there are enough combinations (cygwin vs. MinGW, 32- vs. 64-bit, multiple MinGW installed versions, file locations, statically vs. dynamically linked) that it's best to leave this up to the developer to handle for their specific case.

I'd rather not add a solution that only works for some set ups.

Reply:
Ok. So, we can cancel this PR.
You may mention that tool comes already installed at readme.

Reply:
Thank you. Added a note in the readme in 2f63c613dffe9a7cff73f1898366a2386da92cb0.